cirq-google/cirq_google/api/v2/sweeps.py

```diff
@@ -55,14 +56,18 @@ def _recover_sweep_const(const_pb: run_context_pb2.ConstValue) -> Any:
 
 
 def sweep_to_proto(
-    sweep: cirq.Sweep, *, out: Optional[run_context_pb2.Sweep] = None
+    sweep: cirq.Sweep,
+    *,
+    out: Optional[run_context_pb2.Sweep] = None,
+    func: Callable[..., cirq.Sweep] | None = None,
```

Comment:
rename to something more informative like`sweep_transformer` and make the signature highlight that that it excepts either Linspace or Points 

also make the default value be `lambda x: x` ... this way you don't have to handle `None`